.codecov.yml

```diff
@@ -0,0 +1,25 @@
+coverage:
```

Comment:
I'm not convinced this file actually works -- maybe it needs to be on develop to take effect?